### ISSUE TITLE
Vue-Standalone-Version throws error

### ISSUE THREAD

**Issue by Steviey** [appcomponents/material-components#42]
Chreated with the official webpack template (Vue-Standalone-Version)...
[Vue warn]: You are using the runtime-only build of Vue where the template option is not available. Either pre-compile the templates into render functions, or use the compiler-included build. (found in component <Button>)

**Comment by oskarhane** [comment]
My guess is that this lib isn't compatible with Vuejs 2.

**Comment by alphaelf** [comment]
yes, webpack use vue.common.js as vue core script, if you change it to use vue.js, it's work fine
